src/hooks/sdk/docs/useDeviceLabelTriggerStatus.mdx

```diff
@@ -0,0 +1,57 @@
+<Meta title="SDK Hooks/useDevicePermissionStatus" />
+
+# useDeviceLabelTriggerStatus
+
+The `useDeviceLabelTriggerStatus` hook returns the `DeviceLabelTrigger` status from the enum below.
+
+It tracks if application has triggered the `DeviceLabelTrigger` to request device permission from browser, and if the request is granted or denied.
```

Comment:
I think we want to explain what each of these means in more details, esp `UNTRIGGERED` which can means either it is not triggered yet or the labels are already there so no need to trigger.

Reply:
>esp UNTRIGGERED which can means either it is not triggered yet or the labels are already there so no need to trigger.

Yeah, this is why I rename it from `unset` to `untriggered`. `unset` is very confusing and `untriggered` is the actual status.